javascript/javascript1/week4/characterfrequency.js

```diff
@@ -0,0 +1,20 @@
+function getCharacterFrequencies(incomingWord) {
+  const result = {};
+
+  for (let i = 0; i < incomingWord.length; i++) {
+    let character = incomingWord.charAt(i);
```

Comment:
incomingWord[i] is clearer

Reply:
function getCharacterFrequencies(incomingWord) {
  const result = {};

  for (let i = 0; i < incomingWord.length; i++) {
    let character = incomingWord[i];

    console.log(character);

    if (result[character]) {
      result[character]++;
    } else {
      result[character] = 1;
    }
  }

  return result;
}

const charObject = getCharacterFrequencies("happy");
console.log(charObject);

Reply:
corrected..